### PR TITLE
Add Counting Sort in Kotlin

### LINKED ISSUE #2145: Counting Sort
- [x] C
- [x] C++
- [x] C#
- [x] CoffeeScript
- [ ] Dart 
- [x] Go - @m-code12 
- [x] Java
- [x] JS
- [x] Kotlin - @sandhyabhan
- [x] Php
- [x] Python
- [x] Readme
- [x] Ruby
- [ ] Rust
- [ ] TypeScript - @shivamgupta57121

### PR DESCRIPTION
**Pull request by sandhyabhan**
Fixes #2145 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Add Counting Sort in Kotlin

#### Languages Used:
- Kotlin

#### Files Added:
- Counting Sort in Kotlin

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Counting_Sort/Counting_Sort.kt [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+   	{
+       	    input[i] = read.nextLine().toInt()
+        }
+
+        sort(input);
+        for(i in 0 until arrSize)
+	{
+	    println(input[i]); 
+	}
+
```
Comment (singh-shreya6) [inline Counting_Sort/Counting_Sort.kt:70]: Please remove these blank lines

File: Counting_Sort/Counting_Sort.kt [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    	var input = IntArray(arrSize)
+        println("Enter elements")
+        for(i in 0 until arrSize)
+   	{
+       	    input[i] = read.nextLine().toInt()
+        }
+
+        sort(input);
+        for(i in 0 until arrSize)
+	{
```
Comment (singh-shreya6) [inline Counting_Sort/Counting_Sort.kt:67]: The opening brace and closing brace should be in line with f of for

File: Counting_Sort/Counting_Sort.kt [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        for(i in 0 until n)
+	{
+	    input[i] = output[i]; 
+	}
+    }
+
+    // Main Function
+    fun main()
+    {
+        var read = Scanner(System.`in`)
```
Comment (singh-shreya6) [inline Counting_Sort/Counting_Sort.kt:55]: Please follow 4 space indentation.

File: Counting_Sort/Counting_Sort.kt [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+       
+        for(i in 0 until n)
+        {
+            output[count_array[input[i] - min] - 1] = input[i];
+            count_array[input[i] - min]--;
+        }
+	// Copy the output array to input, so that input now contains sorted values
+        for(i in 0 until n)
+	{
```
Comment (singh-shreya6) [inline Counting_Sort/Counting_Sort.kt:47]: Please correct the indentation here.

**Comment by sandhyabhan** [comment]
I cannot see any indentation issue..Can you please mark it?

**Comment by sandhyabhan** [comment]
Done.

**Pull request merged**